tecken/pytest.ini

```diff
@@ -12,7 +12,9 @@ filterwarnings =
     # django-configurations imports imp;
     # https://github.com/jazzband/django-configurations/issues/190
     ignore:the imp module:DeprecationWarning
-    # django-configurations copies the global settings;
-    # https://github.com/jazzband/django-configurations/issues/233
-    ignore:The DEFAULT_CONTENT_TYPE:django.utils.deprecation.RemovedInDjango30Warning
-    ignore:The FILE_CHARSET:django.utils.deprecation.RemovedInDjango31Warning
+    # dockerflow defines default_app_config in dockerflow.django.__init__ and
+    # shouldn't anymore
+    ignore:.*default_app_config:django.utils.deprecation.RemovedInDjango41Warning
+    # dockerflow passes providing_args as an argument to Signal which it shouldn't
+    # do anymore
+    ignore:.*providing_args:django.utils.deprecation.RemovedInDjango40Warning
```

Comment:
These need to get fixed in dockerflow, but it looks unmaintained. I'll figure out what to do about that another day.